src/analysis/retail/shaman/enhancement/modules/dragonflight/Tier30.tsx

```diff
@@ -21,7 +21,7 @@ const ADDITIONAL_DAMAGE_RATIO = {
   CHAIN_LIGHTNING: 0.2,
 };
 
-export default class Tier30 extends Analyzer {
+export default class extends Analyzer {
```

Comment:
minor nitpick: should prob not drop the class name. the class name shows up in dev tools. without it, you end up with an anonymous class which can be harder to identify

Reply:
Will change, thank you for the suggestion